packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -240,6 +242,13 @@ storiesOf('TableView', module)
       </TableView>
     )
   )
+  .add(
+    // For testing https://github.com/adobe/react-spectrum/issues/1885
+    'controlled selection mode',
```

Comment:
```suggestion
    'swap selection mode',
```
it's not really controlled as it's not a value that onChanges